src/data/python-quiz.ts

```diff
@@ -1,25 +1,13 @@
 const pythonQuiz = [
-   {
-    Question: 
-      "In Python, what is the output of the following code: print(list(range(3, 6)))?",
-    Answer: "[3, 4, 5]",
-    Distractor1: "[3, 4, 5, 6]",
-    Distractor2: "[3, 6]",
-    Distractor3: "[3, 3, 3, 3, 3, 3]",
-    Explanation: 
-      "The list(range(3, 6)) function returns a list of numbers from 3, increments by 1 (by default), and stops at 5 (excluding 6).",
-    Link: "https://www.freecodecamp.org/news/python-range-function-explained-with-code-examples/",
```

Comment:
Why is this question deleted? Please do not remove questions unless a maintainer has approved those changes.

Reply:
It was mistakenly done due to a branch change.
Changes have been reverted.